src/environments/environment.ts

```diff
@@ -24,7 +24,7 @@ export const environment = {
   currencyName: 'Volta Token',
   currencySymbol: 'VT',
   blockExplorerUrl: 'https://volta-explorer.energyweb.org',
-
+  SENTRY_ENVIRONMENT: 'localhost',
```

Comment:
Is there a `localhost` environment for Sentry?

Reply:
Not really. I do not know how this works from Sentry side. Those are environments that are showed to me:
![image](https://user-images.githubusercontent.com/83639710/150783783-bc5ea235-8283-48d6-939c-c137eef86fa5.png)

Should i remove this part? What about develop?

Reply:
Hmm, I think we should perhaps rename `test` to `develop` on Sentry?